js/interface.js

```diff
@@ -178,6 +178,26 @@
     Fliplet.Studio.emit('reload-page-preview');
   });
 
+  Fliplet.Widget.onCancelRequest(function() {
+    console.log(menusPromises)
+    if (menusPromises) {
+      for (var key in menusPromises) {
```

Comment:
is `for in` supported on all browsers?

Reply:
@squallstar I suppose so

<img width="1058" alt="screen shot 2018-01-19 at 18 29 17" src="https://user-images.githubusercontent.com/7046481/35165558-afe57176-fd46-11e7-86bf-28cf136bcc4b.png">

Reply:
IE6, what a time to be alive!